packages/lesswrong/server/cronUtil.ts

```diff
@@ -53,3 +55,19 @@ export function startSyncedCron() {
 onStartup(function() {
   startSyncedCron();
 });
+
+async function clearOldCronHistories() {
+  const ONE_DAY = 24*60*60*1000;
```

Comment:
I think a day is too short - I think it wouldn't be surprising if we added a weekly job. At least I'd like to have a few days of these for debugging purposes. But I don't feel strongly about this so nbd.

Reply:
Makes sense. I just double checked and the old mongo version was set actually set to two days. I'll set it to a week to be safe.